Makefile

```diff
@@ -272,8 +272,8 @@ release-operator: push-image push-bundle-image push-index-image
 .PHONY: prepare-operatorhub-pr
 ## prepare files for OperatorHub PR
 ## use this target when the operator needs to be released as upstream operator
-prepare-operatorhub-pr:
-	./hack/prepare-operatorhub-pr.sh $(VERSION) $(OPERATOR_BUNDLE_IMAGE_REF)
+prepare-operatorhub-pr: yq
+	PATH=$(PATH):bin ./hack/prepare-operatorhub-pr.sh $(VERSION) $(OPERATOR_BUNDLE_IMAGE_REF)
```

Comment:
if you want to be sure that given version of `yq` is used, then it should placed in path before all others. Also, the path to it should be absolute:

```
PATH=$(PWD)/bin:$PATH
```

Reply:
Good point. Thanks @pedjak!